runtime/integration-tests/src/envs/fudge_env.rs

```diff
@@ -46,8 +46,6 @@ impl<T: Runtime + FudgeSupport> Env<T> for FudgeEnv<T> {
 		parachain_storage: Storage,
 		sibling_storage: Storage,
 	) -> Self {
-		crate::utils::logs::init_logs();
```

Comment:
I wanted to remove this because I really think it does not offer us anything to have it by default. Most errors are pallet_timestampt errors or migrations errors that do not affect the test case, offuscating the CI results and local tests.

99% of the test case failures are not something "found" in the logs. If the test requires more debugging, we can always enable it again with...